babel.config.js

```diff
@@ -12,6 +13,38 @@ const targets = isBrowser
 	? { browsers: [ 'last 2 versions', 'Safari >= 10', 'iOS >= 10', 'ie >= 11' ] }
 	: { node: 'current' };
 
+const extensionOverrides = [
+	[
+		'@wordpress/import-jsx-pragma',
+		{
+			scopeVariable: 'createElement',
+			source: '@wordpress/element',
+			isDefault: false,
+		},
+	],
+	[
+		'@babel/transform-react-jsx',
+		{
+			pragma: 'createElement',
+		},
+	],
+];
+
+// The output directory is set when SDK runs, so we only set the POT generator override in that case.
+if ( outputDir ) {
```

Comment:
Even with the comment, I would still prefer the code to tell this story better by renaming the variable something less generic. Not a biggie tho if you decide to leave it like this!